MekHQ/src/mekhq/campaign/mission/ScenarioForceTemplate.java

```diff
@@ -260,6 +260,7 @@ public ScenarioForceTemplate(int forceAlignment, int generationMethod, double fo
         this.destinationZone = destinationZone;
         this.retreatThreshold = retreatThreshold;
         this.allowedUnitType = allowedUnitType;
+        deploymentZones = new ArrayList<>();
```

Comment:
why not: `this.deploymentZones = new ArrayList<>(deploymentZones);`

Reply:
Originally, I did the Collections.copy in response to a code review comment, but I can do it this way too, as I don't see a practical difference.